client/cli/src/arg_enums.rs

```diff
@@ -201,8 +201,10 @@ pub enum Database {
 	/// ParityDb. <https://github.com/paritytech/parity-db/>
 	ParityDb,
 	/// Detect whether there is an existing database. Use it, if there is, if not, create new
-	/// instance of paritydb
+	/// instance of RocksDb
```

Comment:
Isn't it parity db ?
(that is what I got when testing the polkadot parity-db pr, actually I did observe an artifact of rocksdb but it was not used)

Reply:
You are right. I though "Auto" is now the default CLI setting and it defaults to paritydb. But apparently "RocksDb" is still  default.